qemu/tests/win_irq_check.py

```diff
@@ -81,6 +82,10 @@ def set_msi_fguest(enable=True):
     session = utils_test.qemu.windrv_check_running_verifier(session, vm,
                                                             test, driver_verifier,
                                                             timeout)
+    if check_vectors:
+        if int(get_vectors_fqtree()) == int(params["num_queues"]) + 3:
+            test.log.info("Device vectors as expected")
+            return
```

Comment:
Shall we raise the test.fail? `if int(get_vectors_fqtree()) != int(params["num_queues"]) + 3`